### PR TITLE
Use consistent variable name case.

### PR DESCRIPTION
**Pull request by nftw**

### CODE REVIEW

**Comment by bishal-pdMSFT** [comment]
@nftw please update task patch version in task.json

**Comment by nftw** [comment]
@bishal-pdMSFT done

File: Tasks/ServiceFabricDeploy/task.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
     ],
     "version": {
         "Major": 1,
-        "Minor": 2,
-        "Patch": 0
+        "Minor": 4,
+        "Patch": 7
```
Comment (sachinma) [inline Tasks/ServiceFabricDeploy/task.json]: If you build the task - task.loc.json will have the similar change to update the version.

**Comment by nftw** [comment]
@sachinma done

**Pull request merged**